patches/0006-Revert-WAF-schema-changes.patch

```diff
@@ -6,7 +6,7 @@ Subject: [PATCH] Revert WAF schema changes
 - This causes far too many types to be generated downstream.
```

Comment:
Is this normal? I didn't change anything here.

Reply:
I think GitHub's syntax highlighting makes these patches really difficult to look at. It's highlighting it as red, but you didn't actually change anything... the patch just happens to remove something. So you can ignore the stuff that doesn't also have the full-line-width background red color.